README.md

```diff
@@ -43,18 +42,17 @@
 
 ### Key Features <a name="key-features"></a>
 
-
-- **Run a program using the command line.**
-- **Use Ruby syntax for basic programming operations.**
-- **Use Ruby to solve basic coding challenges.**
+-   **Run a program using the command line.**
+-   **Use Ruby syntax for basic programming operations.**
+-   **Use Ruby to solve basic coding challenges.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [N/A]()
+-   [N/A]()
```

Comment:
- [ ] Kindly fix this broken link or you remove the live demo section if there is no live deployment at the moment. 👍